src/main/js/util/url.js

```diff
@@ -1,9 +1,21 @@
 module.exports = {
+    getJobUrl: function (launchUrl, jobName) {
+        // in case we have multiBranch projects or we are using folder, we need to calcultate the path
+        const nameArray = jobName.split('/');
+        var jobUrl = launchUrl;
+        nameArray.map(function (item) {
+            jobUrl += 'job/' + item + '/';
+        });
+        return jobUrl;
+    },
     makeRelative: function(url) {
         return url.indexOf("/") ?
             url.substr(1) :
             url;
     },
+    configureBuildExecutor: function () {
+        return '/computer/(master)/configure';
```

Comment:
I do not know they appear on all my classic jenkins installation when trying to configure nodes (in AT and RUN) I guess that is the official url for that, but you would know better ;)